.gitignore

```diff
@@ -5,3 +5,4 @@ coverage
 src/pipe.min.js
 jsconfig.json
 .vscode
+.idea
```

Comment:
My personal opinion is that should live in your personal config (the same actually goes for .vscode, but that somehow slipped through)